pshtt/utils.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```

Comment:
I think this new hashbang line is not necessary, and could be confusing. (I think the file is also not executable, so it won't necessarily be usable without that anyway.)